handlers/settings.go

```diff
@@ -148,7 +148,7 @@ func bulkImportHandler(w http.ResponseWriter, r *http.Request) {
 // commaToService will convert a CSV comma delimited string slice to a Service type
 // this function is used for the bulk import services feature
 func commaToService(s []string) (*types.Service, error) {
-	if len(s) != 16 {
+	if len(s) != 17 {
 		err := fmt.Errorf("does not have the expected amount of %v columns for a service", 16)
```

Comment:
@hunterlong I think this `16` needs to be updated to `17`

Reply:
`err := fmt.Errorf("does not have the expected amount of %v columns for a service", 17)`